core/src/syn/v1/part/mod.rs

```diff
@@ -67,10 +67,21 @@ pub fn changefeed(i: &str) -> IResult<&str, ChangeFeed> {
 	let (i, _) = tag_no_case("CHANGEFEED")(i)?;
 	let (i, _) = shouldbespace(i)?;
 	let (i, v) = cut(duration)(i)?;
+
+	let (i, opt) = opt(terminated(shouldbespace, tag_no_case("INCLUDE")))(i)?;
+	let mut should_store = false;
+	if opt.is_some() {
+		let (i, _) = shouldbespace(i)?;
+		let (_i, v) = tag_no_case("ORIGINAL")(i)?;
+		if v == "ORIGINAL" {
+			should_store = true;
+		}
+	}
 	Ok((
 		i,
 		ChangeFeed {
 			expiry: v.0,
+			store_original: should_store,
```

Comment:
I don't get the parser here, the behaviour seems weird. Do you want to support an `INCLUDE` keyword without any keyword following it? Also the comparison seems a bit strange, this will cause `INCLUDE original` (different casing) to parse but then not set the store_original variable to true.

The suggestion below requires that after `INCLUDE` you need to parse `ORIGINAL` or parsing fails.
```suggestion

	let (i, opt) = opt(|i| {
		let (i, _) = shouldbespace(i)?;
		let (i, _) = tag_no_case("INCLUDE")?;
		let (i, _) = shouldbespace(i)?;
		tag_no_case("ORIGINAL")
	})(i)?;

	Ok((
		i,
		ChangeFeed {
			expiry: v.0,
			store_original: opt.is_some(),

```